File: llm_generated_code/ChatGPT/CPP/Level1/Q29.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--) {
        int n;
        cin >> n;
        vector<int> a(n);
        for (int i = 0; i < n; ++i) {
            cin >> a[i];
        }
        sort(a.begin(), a.end());
        bool isStrictlyIncreasing = true;
        for (int i = 1; i < n; ++i) {
            if (a[i] <= a[i - 1]) {
                isStrictlyIncreasing = false;
                break;
            }
        }
        cout << (isStrictlyIncreasing ? "YES" : "NO") << endl;
    }
    return 0;
}

File: llm_generated_code/Gemini/Python/Level1/Q29.py
def is_strictly_increasing(arr):
    arr.sort()
    for i in range(1, len(arr)):
        if arr[i] <= arr[i - 1]:
            return False
    return True

def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = list(map(int, input().split()))
        print("YES" if is_strictly_increasing(arr) else "NO")

if __name__ == "__main__":
    main()
